pkgs/top-level/aliases.nix

```
@@ -2695,6 +2695,7 @@ mapAliases {
   xmlroff = throw "'xmlroff' has been removed as it is unmaintained and broken"; # Added 2025-05-18
   xmr-stak = throw "xmr-stak has been removed from nixpkgs because it was broken"; # Added 2024-07-15
   xmake-core-sv = throw "'xmake-core-sv' has been removed, use 'libsv' instead"; # Added 2024-10-10
+  xo = throw "Use 'dbtpl' instead of 'xo'"; # Added 2025-09-28
```

Since it is exactly the same, this shouldn't need manual intervention, and there's no need to throw:
```suggestion
  xo = lib.warnOnInstantiate "'xo' has been renamed to 'dbtpl'" dbtpl; # Added 2025-09-28
```?

---

Do I need to create a new package to handle the issue of changing the bin name? @mdaniels5757

---

Oh, right! Disregard.